sdks/python/setup.py

```diff
@@ -62,6 +62,7 @@ def get_version():
     'oauth2client>=2.0.1',
     'protorpc>=0.9.1',
     'python-gflags>=2.0',
+    'python-snappy>=0.5',
```

Comment:
Unfortunately, it is not possible to specify binary dependencies in in pip packages (https://docs.python.org/dev/installing/#install-scientific-python-packages).  The user has to install them beforehand, before installing this package.  So unless the code is written in such a way as to work in both cases (i.e. with or without the package), it would be unsafe to have this dependency.